android/fakedapp/src/main/java/com/solana/mobilewalletadapter/fakedapp/MainViewModel.kt

```diff
@@ -181,10 +182,8 @@ class MainViewModel(application: Application) : AndroidViewModel(application) {
             if (!authorized) {
                 return@localAssociateAndExecute null
             }
-            val messages = Array(numMessages) {
-                Random.nextBytes(16384)
-            }
-            doSignMessages(client, messages, arrayOf(_uiState.value.publicKey!!))
+            val message = "To avoid digital dognappers, sign below to authenticate with CryptoCorgis.".toByteArray(Charsets.UTF_8)
+            doSignMessages(client, arrayOf(message), arrayOf(_uiState.value.publicKey!!))
```

Comment:
This change will send only 1 message, irrespective of the value of the `numMessages` parameter. What if instead we did:
```
            val messages = Array(numMessages) { i ->
                "To avoid digital dognappers, sign below to authenticate with CryptoCorgis. [$i]".toByteArray(Charsets.UTF_8)
            }
            doSignMessages(client, messages, arrayOf(_uiState.value.publicKey!!))
```

This will append the message index to the string, ensuring that each message is unique (and thus that each signature will be unique as well)